src/consumer.rs

```diff
@@ -61,6 +62,44 @@ pub struct ConsumerOptions {
     pub initial_position: InitialPosition,
 }
 
+impl ConsumerOptions {
```

Comment:
those methods could be proposed in a separate pull request

Reply:
That may be true, but they are usefull in the reader examples, while remaining optional and leaving backward compatibility. It would be as practical to just deliver them in this PR.

Reply:
ok